src/project_variables.rs

```diff
@@ -281,6 +290,16 @@ fn try_key_value_into_slot(
                 regex,
             }),
         },
+        SupportedVarType::Array => VarInfo::Array {
+            entry: Box::new(ArrayEntry {
+                default: if let Some(SupportedVarValue::Array(value)) = default_choice {
+                    Some(value)
+                } else {
+                    None
+                },
+                choices: choices.unwrap(),
```

Comment:
Would be nice to avoid unwraps

Reply:
Good point, I used `unwrap_or_default` here which should give an empty `vec` which I think is fine, but let me know if you want a different solution.